dev/src/main/java/External/CashierDesk.java

```diff
@@ -41,15 +40,21 @@ public void turnOn() throws Exception {
                     break;
                 }
                 case 2: {
-                    displayStores();
+                    if (controllerRef.getStores().size() > 0)
+                        displayStores();
+                    else
+                        this.out.println("No store in the system");
```

Comment:
שולי אבל אנגלית מוזרה קצת הייתי כותב:
There aren't any stores in the system.

Reply:
כנל על ספקים פה למטה

Reply:
Done